netty/src/main/java/io/grpc/transport/netty/NettyChannelBuilder.java

```diff
@@ -118,9 +118,8 @@ public NettyChannelBuilder sslContext(SslContext sslContext) {
   }
 
   /**
-   * Sets the HTTP/2 connection window used for the transport. If not called, uses the default
-   * initial window size from the HTTP/2 specification, as provided by Netty (
-   * {@link #DEFAULT_CONNECTION_WINDOW_SIZE}).
+   * Sets the HTTP/2 connection flow control window. If not called, the default value
```

Comment:
The previous comment talked about the HTTP/2 spec and Netty just seemed wrong to me?